uberenv.py

```diff
@@ -455,30 +469,45 @@ def clean_build(self):
                         res = sexe(unist_cmd, echo=True)
 
     def show_info(self):
-        spec_cmd = "spack/bin/spack spec " + self.pkg_name + self.opts["spec"]
-        return sexe(spec_cmd, echo=True)
+        spec_cmd = "spack/bin/spack spec -IL " + self.pkg_name + self.opts["spec"]
+
+        res, out = sexe(spec_cmd, ret_output=True, echo=True)
+        print(out)
+
+        for line in out.split("\n"):
```

Comment:
Maybe add a comment above the `for` line to say what this is doing ?
Looks like it is looking for duplicates?

Reply:
Done